example/Router.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Equip\Queue\Exception\HandlerException;
+use Equip\Queue\Exception\RouterException;
+use Equip\Queue\RouterInterface;
+
+class Router implements RouterInterface
```

Comment:
I didn't plan on providing an implementation, but this one turned out alright. I'll add it.